src/spells.cpp

```diff
@@ -572,11 +572,8 @@ bool Spell::configureSpell(const pugi::xml_node& node)
 
 		int32_t vocationId = g_vocations.getVocationId(attr.as_string());
 		if (vocationId != -1) {
-			vocSpellMap[vocationId] = true;
-			int32_t promotedVocation = g_vocations.getPromotedVocation(vocationId);
-			if (promotedVocation != VOCATION_NONE) {
-				vocSpellMap[promotedVocation] = true;
-			}
+			attr = vocationNode.attribute("showInDescription");
+			vocSpellMap[vocationId] = not attr or attr.as_bool();
```

Comment:
Yes, but he didn't change it. It's not about whether or not they're defined, but about being consistent. We're using ! and || everywhere else, it's just confusing to suddenly have a line where we use different syntax.